WebApp/backend/QuizMaster.API.Gatewway/Controllers/MediaGatewayController.cs

```diff
@@ -0,0 +1,205 @@
+﻿using Grpc.Core;
+using Grpc.Net.Client;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.StaticFiles;
+using Newtonsoft.Json;
+using QuizMaster.API.Media.Models;
+using QuizMaster.API.Media.Proto;
+using QuizMaster.API.Media.Utility;
+
+namespace QuizMaster.API.Gateway.Controllers
+{
+    [ApiController]
+    [Route("gateway/api/media")]
+    public class MediaGatewayController : Controller
+    {
+        private readonly GrpcChannel _channel;
+        private readonly MediaService.MediaServiceClient _channelClient;
+        private readonly ILogger _logger;
+        private readonly IWebHostEnvironment _webHostEnvironment;
+
+        public MediaGatewayController(ILogger<MediaGatewayController> logger, IWebHostEnvironment webHostEnvironment)
+        {
+            _channel = GrpcChannel.ForAddress("https://localhost:7197");
```

Comment:
I'll suggest instead of hardcoding the localhost in line 23 `_channel = GrpcChannel.ForAddress("https://localhost:7197");`
put it inside a configuration file like for example: AppSettings.MICROSERVICE_MEDIA_HOST